README.md

```diff
@@ -95,10 +95,31 @@ See [contributing documentation](./CONTRIBUTING.md)
 See [full documentation](docs/README.md)
 
 
+# Useful tips
+
+## Updating kuzzle's containers
+
+When you already have installed an old version of kuzzle, don't forget to update kuzzle's containers with:
+
+```
+    $ docker-compose -f <docker-compose-file.yml> pull 
+```
+
+## Updating kuzzle's dependencies 
+
+To ensure that Kuzzle's dependencies are up-to-date, go into a Kuzzle container and run:
+
+```
+    (host)      $ docker exec -ti <docker-compose-file.yml> bash
+    (container) $ npm install
+    (container) $ bin/kuzzle.js install
+```
```

Comment:
better for me: run the command directly without log-in into the container:
$ docker exec -ti <docker-compose-file.yml>  npm install
$ docker exec -ti <docker-compose-file.yml>  bin/kuzzle.js install